servicemanager/service/smplayservice.py

```diff
@@ -117,24 +119,38 @@ def start_from_binary(self):
                 print b.fail + "ERROR: could not start '" + self.service_name + "' " + b.endc
             return popen_output.pid
 
-    def _unzip_play_application(self):
+    def _unpack_play_application(self, extension):
```

Comment:
ah yes. kinda. I meant using [os.path.splitext](https://docs.python.org/2/library/os.path.html#os.path.splitext) to grab the extension.

I should've left this comment against [this line](https://github.com/hmrc/service-manager/pull/34/files#diff-66a883b6dfd002b95edaafaed230bcffR18) instead